doc/api/code_integrity.md

```diff
@@ -0,0 +1,133 @@
+# Code Integrity
+
+<!--introduced_in=v24.0.0-->
```

Comment:
I think due to the time frame, it will be unlikely to land on v24.0.0. Also I consider it as semver-minor, so it can land in a later release.